core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -661,8 +668,19 @@ public void setVolume(final float volumeLeft, float volumeRight) {
      */
     private void setVolumeSync(float volumeLeft, float volumeRight) {
         playerLock.lock();
-        mediaPlayer.setVolume(volumeLeft, volumeRight);
-        Log.d(TAG, "Media player volume was set to " + volumeLeft + " " + volumeRight);
+        if (media != null && EnumSet.of(MediaType.AUDIO, MediaType.VIDEO).contains(media.getMediaType())) {
```

Comment:
I am wondering if this check can just be omitted. We only have audio and video feeds.

Reply:
Well `MediaType` includes `UNKNOWN`, so technically there could be another type for which I don't want to adapt volume. Should I drop the check anyway?

Reply:
I think the rest of the code will take care of the unknown cases automatically. Not sure where unknown media types are actually used.

Reply:
I have done so.